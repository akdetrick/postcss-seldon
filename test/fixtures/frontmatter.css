/**doc
---
name: Test Name
id: 4
---

Just some good old markdown here
*/
