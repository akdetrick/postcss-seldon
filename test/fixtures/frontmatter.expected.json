{
  "meta": {
    "name": "Test Name",
    "id": 4
  },
  "markdown": "\nJust some good old markdown here",
  "html": "<p>Just some good old markdown here</p>\n"
}
